com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -1979,6 +1979,9 @@ public int areaLightEmissiveMeshLayer
             }
         }
 
+        public delegate Matrix4x4 CustomViewCallback(Matrix4x4 lightLocalToWorldMatrix);
+        public CustomViewCallback CustomViewCallbackEvent;
```

Comment:
please add documentation for public API :)

Reply:
+ add a changelog

Reply:
Added :)